src/audio/kpb.c

```diff
@@ -982,9 +982,6 @@ static void kpb_init_draining(struct comp_dev *dev, struct kpb_client *cli)
 		/* Pause selector copy. */
 		kpb->sel_sink->sink->state = COMP_STATE_PAUSED;
 
-		/* Disable system agent during draining */
```

Comment:
Since all `sa_()` API calls are removed, is *sof/lib/agent.h* include still required?

Reply:
@mmaka1 Yes, good catch. Will fix.